jsk_pcl_ros/CMakeLists.txt

```diff
@@ -7,21 +7,46 @@ project(jsk_pcl_ros)
 set(PCL_MSGS pcl_msgs) ## hydro and later
 
 find_package(catkin REQUIRED COMPONENTS
-  jsk_recognition_utils jsk_pcl_ros_utils
-  dynamic_reconfigure pcl_ros nodelet message_generation genmsg
-  ${PCL_MSGS} sensor_msgs geometry_msgs jsk_recognition_msgs
-  eigen_conversions tf_conversions tf2_ros tf
-  image_transport nodelet cv_bridge
-  jsk_topic_tools
+  cv_bridge
+  diagnostic_msgs
+  diagnostic_updater
+  dynamic_reconfigure
+  eigen_conversions
+  genmsg
+  geometry_msgs
   image_geometry
-  jsk_footstep_msgs
+  image_transport
+  image_view2
   interactive_markers
+  jsk_footstep_msgs
+  jsk_pcl_ros_utils
+  jsk_recognition_msgs
+  jsk_recognition_utils
+  jsk_topic_tools
+  kdl_conversions
+  kdl_parser
   laser_assembler
-  octomap_server
-  octomap_ros
+  message_generation
+  ml_classifiers
+  moveit_core
+  nav_msgs
+  nodelet
   octomap_msgs
-  kdl_parser
-  kdl_conversions
+  octomap_ros
+  octomap_server
+  pcl_conversions
+  ${PCL_MSGS}
+  pcl_ros nodelet
+  rosboost_cfg
+  roscpp_tutorials
+  sensor_msgs
+  std_msgs
```

Comment:
ok, let's if we really need std_msgs
https://github.com/jsk-ros-pkg/jsk_recognition/pull/1846


--
◉ Kei Okada


On Tue, Aug 23, 2016 at 1:03 PM, Kentaro Wada <notifications@github.com>
wrote:

> In jsk_pcl_ros/CMakeLists.txt
> <https://github.com/jsk-ros-pkg/jsk_recognition/pull/1828#discussion_r75799730>
> :
>
> >    octomap_msgs
> > -  kdl_parser
> > -  kdl_conversions
> > +  octomap_ros
> > +  octomap_server
> > +  pcl_conversions
> > +  ${PCL_MSGS}
> > +  pcl_ros nodelet
> > +  rosboost_cfg
> > +  roscpp_tutorials
> > +  sensor_msgs
> > +  std_msgs
>
> Any comments, please?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/jsk-ros-pkg/jsk_recognition/pull/1828/files/5c8d7d04c921a71dee29025a37f9da0e73143e4e..26cb133674dfb49b96f46dda7e38e77d67530d87#r75799730>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAeG3CO88COaqVDIZNFqJRR-R3mbf5Z1ks5qinD2gaJpZM4JeUkQ>
> .
>